index.js

```diff
@@ -10,8 +10,11 @@ var log = require('./lib/logger');
 // note that test files have require('ava')
 require('./lib/test-worker').avaRequired = true;
 
-var opts = JSON.parse(process.argv[2]);
-var runner = new Runner(opts);
+var opts = globals.options;
+var runner = new Runner({
+	serial: Boolean(opts.serial),
+	bail: Boolean(opts.failFast)
```

Comment:
Why do you need to coerce these to booleans? If it's needed it should happen at the source where they're gotten/defined.